File: tests/javascript/third_parties/airbrake/testdata/datatype_in_notify.js
const { Notifier } = require('@airbrake/node');

const airbrake = new Notifier({
  projectId: 42,
  projectKey: 'some-project-key',
  environment: 'PROD',
});

// bearer:expected javascript_third_parties_airbrake
let promise = airbrake.notify("user " + currentUser().emailAddress)
promise.then(() => {})

riskyCode(() => {
  try {
    // something risky
  } catch (err) {
// bearer:expected javascript_third_parties_airbrake
    airbrake.notify({
      error: err,
      params: { user: user.ipAddress },
    });
  }
})

File: tests/ruby/third_parties/airbrake/testdata/datatype_in_notify.rb
# bearer:expected ruby_third_parties_airbrake
Airbrake.notify(user.first_name)

# bearer:expected ruby_third_parties_airbrake
Airbrake.notify('App crashed!', {
  current_user: user.email
})

# bearer:expected ruby_third_parties_airbrake
Airbrake.notify('App crashed') do |notice|
  notice[:params][:email] = customer.email
end